### DISCUSSION TITLE
Why the todo keyword in multiline comment is not highlighted?

### DISCUSSION THREAD

**Discussion by hopezh** [folke/todo-comments.nvim#231] [Q&A] [answered]
As shown below, the highlight color for the custom comment keyword "oat" is not applied in a multiline comment. 

May  I ask how to get the same highlight color for the keywords in multiline comments? 

<img width="1224" alt="Screenshot 2023-10-01 at 10 31 31 PM" src="https://github.com/folke/todo-comments.nvim/assets/23435810/b1ac0312-a0ea-41a4-8aea-2c903905e4f2">

**Comment by bleonard252** [comment, marked as answer]
That's a multiline string (even if it is functionally equivalent to a multiline comment).
